### PR TITLE
Add public doc for cascading inputs in tool

### PR DESCRIPTION
**Pull request by chenslucky**
# Description

Add a public doc to instruct custom through the process of implementing cascading settings for tool inputs.

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [x] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### CODE REVIEW

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+> Note: The "enabled_by_value" in the tool yaml is of list type, implying that a single input can be enabled by multiple values from the dependent input.
+
+## Prerequisites
+To proceed, it's crucial for you to understand the process of developing a tool and generating a tool yaml. For thorough insights and instructions, please refer to [Create and Use Tool Package](create-and-use-tool-package.md).
+
+## Use "enabled_by + enabled_by_value" in tool
+Here we take [this tool](https://github.com/microsoft/promptflow/blob/main/examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_enabled_by_value.py) as example, this tool has three inputs:
+"user_type", "student_id", and "teacher_id".
+The "student_id" and "teacher_id" are enabled by the value of input "user_type". When the value of "user_type" input is "student", the "student_id" input is enabled and displayed.
+When the type of "user_type" input is "teacher", the "teacher_id" input is enabled and displayed.
```
Comment (dorisjoy) [inline docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md:17]: When the value of "user_type" input is "teacher" ?

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Below shows how to support this cascading setting in both tool code and tool yaml.
+
+### Step 1: How to define the rules in the tool
+All inputs will be defined in the tool, allowing you to define your own set of rules to determine which input to use. You need to pay attention to some key points:
+* When an input has the potential to have multiple values, an Enum class such as "UserType" as shown in the following example can be created.
```
Comment (dorisjoy) [inline docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md:22]: When an input should only take on a certain set of fixed values, an Enum class such as "UserType" as shown in the following example can be created.

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+* Within the tool's logic, various inputs may be used depending on the respective "user_type" values.
+
+Here is an example of how you define the rules in your tool:
```
Comment (dorisjoy) [inline docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md:25]: [an example](link to your tool.py)

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    if user_type == UserType.STUDENT:
+        return student_id
+    elif user_type == UserType.TEACHER:
+        return teacher_id
+    else:
+        raise Exception("Invalid user.")
+```
+
+### Step 2: Support "enabled_by_value" in the tool yaml
+Once you have generated a tool yaml, you can incorporate the "enabled_by_value" into it. Here is an example showcasing the use of "enabled_by_value" in the tool yaml:
```
Comment (dorisjoy) [inline docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md:59]: [an example showcasing](link to your tool.yaml)

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+      enabled_by: user_type
+      enabled_by_value: [teacher]
+  module: my_tool_package.tools.tool_with_enabled_by_value
+  name: My Tool with Enabled By Value
+  description: This is my tool with enabled by value
+  type: python
+```
+
+## Use the tool from VSCode Extension
+After you build and share the tool package, you can use your tool from VSCode Extension according to [Create and Use Tool Package](create-and-use-tool-package.md). For instance, when you select a "user_type" with "student" value, only "student_id" input is enabled and displayed for "enabled_by_value" example.
```
Comment (dorisjoy) [inline docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md:88]: After you build and share the tool package, you can use your tool from VSCode Extension according to [Create and Use Tool Package](create-and-use-tool-package.md). Here we use an existing flow to demonstrate the experience, open [this flow](link to your flow) in VS Code extension. Then, when you set the value of input "user_type" to "student", only "student_id" input is enabled and displayed.

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
+# Implementing Cascading Settings for Tool Inputs
```
Comment (DaweiCai) [inline docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md:1]: make the doc file name consistent with the doc title?  how-to-use-cascading-inputs?

Comment (DaweiCai) [inline docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md:1]: use cascading inputs in tool

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+Cascading settings between inputs are frequently used in situations where the selection in one input field determines what subsequent inputs should be shown.
+This approach help in creating a more efficient, user-friendly, and error-free input process.
+This article will guide you through the process of implementing cascading settings for tool inputs.
+
+## Prerequisites
+* Please ensure that your Prompt flow for VS Code is updated to version 1.2.0 or later.
+* It's crucial for you to understand the process of creating your own tool package. For thorough insights and instructions, please refer to [Create and Use Tool Package](create-and-use-tool-package.md).
```
Comment (DaweiCai) [inline docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md]: Not necessary to mention this in prerequisites.

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+## Create custom tool package with cascading settings
```
Comment (DaweiCai) [inline docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md]: ## Create custom tool package with cascading inputs ?

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+    """
+    if user_type == UserType.STUDENT:
+        return student_id
+    elif user_type == UserType.TEACHER:
+        return teacher_id
+    else:
+        raise Exception("Invalid user.")
+```
+
+2. Generate a tool yaml for your tool, referring to [tool_with_enabled_by_value.yaml](https://github.com/microsoft/promptflow/blob/main/examples/tools/tool-package-quickstart/my_tool_package/yamls/tool_with_enabled_by_value.yaml) as an example. The "enabled_by_value" in one input means that this input is enabled by the value of the input referred to in the "enabled_by" attribute.
```
Comment (DaweiCai) [inline docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md]: How to generate the tool yaml? does customer need to manually update the yaml file?

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md (hunk 6 above)
Comment (chenslucky) [reply docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md]: renamed

Comment (chenslucky) [reply docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md]: make the doc file name consistent with the doc title

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md (hunk 8 above)
Comment (chenslucky) [reply docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md]: refine the section name

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md (hunk 7 above)
Comment (chenslucky) [reply docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md]: deleted

File: docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md (hunk 9 above)
Comment (chenslucky) [reply docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md]: refine this part

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
+# Use Cascading Inputs in Tool
+
+Cascading settings between inputs are frequently used in situations where the selection in one input field determines what subsequent inputs should be shown.
+This approach help in creating a more efficient, user-friendly, and error-free input process.
```
Comment (dorisjoy) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:4]: this approach helps ?

**Review by DaweiCai — DISMISSED** [review]

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
+# Use Cascading Inputs in Tool
+
+Cascading settings between inputs are frequently used in situations where a selection in one input field determines what subsequent inputs should be shown. This approach helps create a more efficient, user-friendly, and error-free input process. This article will guide you through the process of implementing cascading settings for tool inputs.
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:3]: Cascading input settings are useful when the value of one input field determines which subsequent inputs are shown. This makes the input process more streamlined, user-friendly, and error-free. This guide will walk through how to create cascading inputs for your tools.

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+
+## Prerequisites
+Please ensure that your [Prompt flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) is updated to version 1.2.0 or later.
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:6]: Please make sure you have the latest version of [Prompt Flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) installed (v1.2.0+).

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+
+
+## Create tool with cascading inputs
+We will provide a hands-on tool example to demonstrate the implementation of cascading settings between inputs within a tool. 
+In this tool, the availability of 'student_id' and 'teacher_id' inputs is determined by the value of the 'user_type' input. 
+Below shows how to support this cascading setting in both tool code and tool yaml.
+
+1. Develop your tool within the def function, referring to [tool_with_cascading_inputs.py](https://github.com/microsoft/promptflow/blob/main/examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_cascading_inputs.py) as an example. You need to pay attention to some key points:
+    * Use the `@tool` decorator to identify the function as a tool.
+    * When an input should only take on a certain set of fixed values, an Enum class such as `UserType` as shown in the following example can be created.
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:16]: Enum class is unnecessary? I think user are free to use string or constant also.

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 14]
```diff
@@ -0,0 +1,136 @@
...
+## Prerequisites
+Please ensure that your [Prompt flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) is updated to version 1.2.0 or later.
+
+
+## Create tool with cascading inputs
+We will provide a hands-on tool example to demonstrate the implementation of cascading settings between inputs within a tool. 
+In this tool, the availability of 'student_id' and 'teacher_id' inputs is determined by the value of the 'user_type' input. 
+Below shows how to support this cascading setting in both tool code and tool yaml.
+
+1. Develop your tool within the def function, referring to [tool_with_cascading_inputs.py](https://github.com/microsoft/promptflow/blob/main/examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_cascading_inputs.py) as an example. You need to pay attention to some key points:
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:14]: 1. Develop the tool function, following the [cascading inputs example](https://github.com/microsoft/promptflow/blob/main/examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_cascading_inputs.py). Key points:

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 15]
```diff
@@ -0,0 +1,136 @@
...
+    * Use the `@tool` decorator to identify the function as a tool.
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:15]: Use the @tool decorator to mark the function as a tool.

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 16]
```diff
@@ -0,0 +1,136 @@
...
+    * When an input should only take on a certain set of fixed values, an Enum class such as `UserType` as shown in the following example can be created.
+    * Within the tool's logic, various inputs may be used depending on the value of the `user_type` input.
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:17]: Conditionally use inputs in the tool logic based on "user_type".

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 17]
```diff
@@ -0,0 +1,136 @@
...
+    """
+    if user_type == UserType.STUDENT:
+        return student_id
+    elif user_type == UserType.TEACHER:
+        return teacher_id
+    else:
+        raise Exception("Invalid user.")
+```
+
+2. Follow the guide [Create and Use Tool Package](create-and-use-tool-package.md) to generate a tool yaml for your tool. Afterwards, manually update this tool yaml to transition from common inputs to cascading inputs.
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:49]: Generate a starting YAML for your tool per the [tool package guide](https://claude.ai/chat/create-and-use-tool-package.md), then update it to enable cascading:

Add "enabled_by" and "enabled_by_value" to control visibility of dependent inputs.
See the [example YAML](https://github.com/microsoft/promptflow/blob/main/examples/tools/tool-package-quickstart/my_tool_package/yamls/tool_with_cascading_inputs.yaml) for reference.
Note: "enabled_by_value" takes a list, allowing multiple values to enable an input.

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 18]
```diff
@@ -0,0 +1,136 @@
...
+      enabled_by_value: [teacher]
+  module: my_tool_package.tools.tool_with_cascading_inputs
+  name: My Tool with Cascading Inputs
+  description: This is my tool with cascading inputs
+  type: python
+```
+> Note: The `enabled_by_value` in the tool yaml is of list type, implying that a single input can be enabled by multiple values from the dependent input.
+
+## Use your tool from VSCode Extension
+Once you've built and shared your tool, you can use your tool from VSCode Extension according to [Create and Use Tool Package](create-and-use-tool-package.md).
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:82]: Once you package and share your tool, you can use it in VS Code per the [tool package guide](https://claude.ai/chat/create-and-use-tool-package.md). We have a [demo flow](https://github.com/microsoft/promptflow/tree/main/examples/tools/use-cases/cascading-inputs-tool-showcase) you can try.

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 19]
```diff
@@ -0,0 +1,136 @@
...
+Here we use [an existing flow](https://github.com/microsoft/promptflow/tree/main/examples/tools/use-cases/cascading-inputs-tool-showcase) to demonstrate the experience.
+
+Before you select the `user_type` input, both `student_id` and `teacher_id` inputs are disabled and hidden.
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:85]: Before selecting a "user_type", the "student_id" and "teacher_id" inputs are hidden. Once you pick the user type, the corresponding input appears.

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 20]
```diff
@@ -0,0 +1,136 @@
...
+![before_user_type_selected.png](../../media/how-to-guides/develop-a-tool/before_user_type_selected.png)
+
+However, once you select the `user_type` input, the corresponding input will be enabled and displayed.
+![after_user_type_selected_with_student.png](../../media/how-to-guides/develop-a-tool/after_user_type_selected_with_student.png)
+![after_user_type_selected_with_teacher.png](../../media/how-to-guides/develop-a-tool/after_user_type_selected_with_teacher.png)
+
+
+
+## FAQ
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:94]: FAQs

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 21]
```diff
@@ -0,0 +1,136 @@
...
+      # This input is enabled by the input "event_type".
+      enabled_by: event_type
+      # This input is enabled when "event_type" is "private".
+      enabled_by_value: [private]
+  module: my_tool_package.tools.tool_with_multi_layer_cascading_inputs
+  name: My Tool with Multi-Layer Cascading Inputs
+  description: This is my tool with multi-layer cascading inputs
+  type: python
+```
+When the tool is run, the inputs will be displayed in a cascading manner. As each input is filled out, any inputs that are dependent on it and have their `enabled_by_value` condition met will be enabled.
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:136]: Inputs will enable in a cascading way based on selections.

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 22]
```diff
@@ -0,0 +1,136 @@
...
+![before_user_type_selected.png](../../media/how-to-guides/develop-a-tool/before_user_type_selected.png)
+
+However, once you select the `user_type` input, the corresponding input will be enabled and displayed.
+![after_user_type_selected_with_student.png](../../media/how-to-guides/develop-a-tool/after_user_type_selected_with_student.png)
+![after_user_type_selected_with_teacher.png](../../media/how-to-guides/develop-a-tool/after_user_type_selected_with_teacher.png)
+
+
+
+## FAQ
+### How to use multi-layer cascading inputs in tool?
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:95]: How do I create multi-layer cascading inputs?

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 23]
```diff
@@ -0,0 +1,136 @@
...
+If you are dealing with multiple levels of cascading inputs, you can effectively manage the dependencies between them by using the `enabled_by` and `enabled_by_value` attributes. Here's a hypothetical YAML example.
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:96]: Here's a hypothetical YAML example. -> For example:

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 24]
```diff
@@ -0,0 +1,136 @@
+# Use Cascading Inputs in Tool
+
+Cascading settings between inputs are frequently used in situations where a selection in one input field determines what subsequent inputs should be shown. This approach helps create a more efficient, user-friendly, and error-free input process. This article will guide you through the process of implementing cascading settings for tool inputs.
+
+## Prerequisites
+Please ensure that your [Prompt flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) is updated to version 1.2.0 or later.
+
+
+## Create tool with cascading inputs
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:9]: a tool

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md [hunk 25]
```diff
@@ -0,0 +1,136 @@
...
+      enabled_by: user_type
+      enabled_by_value: [teacher]
+  module: my_tool_package.tools.tool_with_cascading_inputs
+  name: My Tool with Cascading Inputs
+  description: This is my tool with cascading inputs
+  type: python
+```
+> Note: The `enabled_by_value` in the tool yaml is of list type, implying that a single input can be enabled by multiple values from the dependent input.
+
+## Use your tool from VSCode Extension
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md:81]: the tool

File: docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md (hunk 13 above)
Comment (chenslucky) [reply docs/how-to-guides/develop-a-tool/use-cascading-inputs-in-tool.md]: add more explanation.

File: docs/how-to-guides/develop-a-tool/create-cascading-tool-inputs.md [hunk 26]
```diff
@@ -0,0 +1,136 @@
...
+      enabled_by: user_type
+      enabled_by_value: [teacher]
+  module: my_tool_package.tools.tool_with_cascading_inputs
+  name: My Tool with Cascading Inputs
+  description: This is my tool with cascading inputs
+  type: python
+```
+
+## Use the tool in VS Code
+Once you package and share your tool, you can use it in VS Code per the [tool package guide](create-and-use-tool-package.md). We have a [demo flow](https://github.com/microsoft/promptflow/tree/main/examples/tools/use-cases/cascading-inputs-tool-showcase) you can try.
```
Comment (dorisjoy) [inline docs/how-to-guides/develop-a-tool/create-cascading-tool-inputs.md:85]: "Once you package and share your tool" not a complete sentence

File: docs/how-to-guides/develop-a-tool/create-cascading-tool-inputs.md [hunk 27]
```diff
@@ -0,0 +1,136 @@
...
+## Prerequisites
+Please make sure you have the latest version of [Prompt flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) installed (v1.2.0+).
+
+
+## Create a tool with cascading inputs
+We'll build out an example tool to show how cascading inputs work. The `student_id` and `teacher_id` inputs will be controlled by the value selected for the `user_type` input. Here's how to configure this in the tool code and YAML.
+
+1. Develop the tool function, following the [cascading inputs example](https://github.com/microsoft/promptflow/blob/main/examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_cascading_inputs.py). Key points:
+    * Use the `@tool` decorator to mark the function as a tool.
+    * Define `UserType` as an Enum class, as it accepts only a specific set of fixed values in this example. In fact, you can use other types depending on your own scenarios.
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/create-cascading-tool-inputs.md:14]: delete this line.

File: docs/how-to-guides/develop-a-tool/create-cascading-tool-inputs.md [hunk 28]
```diff
@@ -0,0 +1,136 @@
...
+        return teacher_id
+    else:
+        raise Exception("Invalid user.")
+```
+
+2. Generate a starting YAML for your tool per the [tool package guide](create-and-use-tool-package.md), then update it to enable cascading:
+
+    Add `enabled_by` and `enabled_by_value` to control visibility of dependent inputs. See the [example YAML](https://github.com/microsoft/promptflow/blob/main/examples/tools/tool-package-quickstart/my_tool_package/yamls/tool_with_cascading_inputs.yaml) for reference.
+
+    * The `enabled_by` attribute specifies which input field controls the visibility of the dependent input field.
```
Comment (chjinche) [inline docs/how-to-guides/develop-a-tool/create-cascading-tool-inputs.md:51]: The enabled_by attribute specifies the input field, which must be an enum type, that controls the visibility of the dependent input field.

The enabled_by_value attribute defines the accepted enum values from the enabled_by field that will make this dependent input field visible.

**Review by chjinche — APPROVED** [review]

**Review by dorisjoy — APPROVED** [review]

**Pull request merged**